src/components/Dropdown/index.tsx

```diff
@@ -25,13 +27,21 @@ const renderOptions = (options: OptionItem[]) => options.map(createOptions);
 
 const Dropdown: React.FC<DropdownProps> = (props) => {
 
-  const { className, name, options, title } = props;
+  const { action, className, name, options, title } = props;
   const [dropdownValue, setDropdownValue] = React.useState<string | number>();
   const classes = dropdownStyles();
   const rootClass = classNames(classes.root, className);
 
+  const getDocumentType = (id: string) => {
+    const type = options.filter(opt => opt.id === id);
+    if (!type || !type.length) return '';
+    return type[0].value;
+  }
+
   const handleChange = (event: React.ChangeEvent<{ value: any }>) => { 
-    setDropdownValue(event.target.value);
+    const value = event.target.value;
+    setDropdownValue(value);
+    action(getDocumentType(value));
```

Comment:
This action should be enough to update the redux state and, the prop this dropdown relies on to display the selected option, should be mapped from the state to a prop

Reply:
Gonna do this change on the next PR.